src/Dialog/withResponsiveFullScreen.js

```diff
@@ -18,18 +18,12 @@ function withResponsiveFullScreen(options: Options = { breakpoint: 'sm' }) {
   return (BaseDialog: Dialog) => {
     const factory = createEagerFactory(BaseDialog);
 
-    const ResponsiveFullScreen = (props: { width: string }) => {
-      const { width } = props;
-
-      const fullScreen = isWidthDown(breakpoint, width);
-
-      const dialogProps = {
-        fullScreen,
+    function ResponsiveFullScreen(props: { width: string }) {
```

Comment:
No real motivation, I was in that mood:
1. Using named function over anonymous one everywhere possible
2. Removing intermediary variables